numpy/core/arrayprint.py

```diff
@@ -51,24 +51,35 @@
     _MAXINT = sys.maxint
     _MININT = -sys.maxint - 1
 
-def product(x, y):
-    return x*y
+_format_options = {
+    'edgeitems': 3,  # repr N leading and trailing items of each dimension
+    'threshold': 1000,  # total items > triggers array summarization
+    'precision': 8,
+    'suppress': False,
```

Comment:
These names (`_float_output_suppress_small` and `_float_precision`) used to be more descriptive - can you add comments that these only effect floats, and that suppress is something about small values?